todoie/Bindable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Bindable.swift
+//  todoie
+//
+//  Created by Mustafa Khalil on 1/2/19.
+//  Copyright © 2019 Aurora. All rights reserved.
+//
+
+import Foundation
+
+class Bindable<T>{
```

Comment:
what is this used for exactly? How does it bind the viewcontroller with viewmodel?

Reply:
@suhabaobaid read the following comments